test/cpp_extensions/libtorch_agnostic_extension/test/test_libtorch_agnostic.py

```diff
@@ -39,6 +49,8 @@ def test_slow_sgd(self, device):
 
     @onlyCUDA
     def test_identity_does_not_hog_memory(self, device):
+        import libtorch_agnostic
```

Comment:
Why not put this in line 22?

Reply:
Putting the import there doesn't import to global scope, so the test functions still won't be able to see it. There's a hacky way to avoid updating the test functions:

```python
globals()["libtorch_agnostic"] = importlib.import_module("libtorch_agnostic")
```

Reply:
nawwwww what u have rn is more explicit i agree